Source/Eto/Drawing/PaddingConverter.cs

```diff
@@ -12,6 +12,11 @@ namespace Eto.Drawing
 	public class PaddingConverter : TypeConverter
 	{
 		/// <summary>
+		/// The character to split up the string which will be converted
+		/// </summary>
+		private readonly string[] StringSplitter = new string[1] { "," };
```

Comment:
Hm, sorry thought this was more of a academic discussion and pulled before you made the change.